### PR TITLE
feat: Implement a command to delete a note

### PR DESCRIPTION
**Pull request by ArzamastsevOleksandr**

### CODE REVIEW

File: src/main/java/com/booster/cliclient/command/handler/DeleteNoteCommandHandler.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        Request request = new Request.Builder()
+                .url("http://localhost:8081/note/" + id)
+                .delete()
+                .build();
+
+        try (Response response = okHttpClient.newCall(request).execute()) {
+            if (!response.isSuccessful()) {
+                outputWriter.print("Error occurred");
```
Comment (LarysaDoroshenko) [inline src/main/java/com/booster/cliclient/command/handler/DeleteNoteCommandHandler.java:39]: In the previous PR, you extended a message with a response code. Do we need the same here?

Comment (LarysaDoroshenko) [reply src/main/java/com/booster/cliclient/command/handler/DeleteNoteCommandHandler.java:39]: And you should use println method instead of print

**Comment by LarysaDoroshenko** [comment]
Check tests locally. Now test for the help command fails, as there is no line for the 'delete note' command.

File: src/main/java/com/booster/cliclient/command/handler/DeleteNoteCommandHandler.java (hunk 1 above)
Comment (ArzamastsevOleksandr) [reply src/main/java/com/booster/cliclient/command/handler/DeleteNoteCommandHandler.java:39]: We use `print` in all handlers.

I am not sure about the `response code` for now. We will refactor this part and get rid of the duplication soon

**Review by LarysaDoroshenko — APPROVED** [review]

**Pull request merged**